apps/web/src/components/tx-flow/flows/TokenTransfer/RecipientRow/index.tsx

```diff
@@ -72,18 +72,11 @@ export const RecipientRow = ({ fieldArray, removable = true, remove, disableSpen
 
   const maxAmount = isSpendingLimitType && totalAmount > spendingLimitAmount ? spendingLimitAmount : totalAmount
 
-  const deps = useMemo(
-    () =>
-      recipients.map((_, index) =>
-        getFieldName(TokenTransferFields.amount, { name: MultiTokenTransferFields.recipients, index }),
-      ),
-    [recipients],
-  )
-
   const onRemove = useCallback(() => {
     remove?.(fieldArray.index)
-    trigger(deps)
-  }, [remove, fieldArray.index, trigger, deps])
+    clearErrors(MultiTokenTransferFields.recipients)
```

Comment:
Nit: you don't need to call `clearErrors` before `trigger`.

Reply:
Adjusted